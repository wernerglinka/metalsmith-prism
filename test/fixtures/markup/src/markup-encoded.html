<code class="language-markup">
&lt;html&gt;
</code>
