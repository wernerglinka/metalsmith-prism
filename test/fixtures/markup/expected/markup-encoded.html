<html><head></head><body><code class="language-markup">
  <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>html</span><span class="token punctuation">&gt;</span></span>
</code>
</body></html>
